tests/test_webhook.py

```diff
@@ -144,7 +145,7 @@ def poll_job(self, resp_data):
     #         time.sleep(3)
     #     tm = Template(json_dumps(pc))
     #     rv = self.server.post(
-    #         URL_PREFIX + "/locations/nc_spm_08/processing_async_export",
+    #         URL_PREFIX + "/projects/nc_spm_08/processing_async_export",
```

Comment:
Maybe add test for both until GRASS GIS 9 ?

Reply:
Ist schon geändert. War leider noch nicht commited, da ich die Tests verschoben hatte